requirements.txt

```diff
@@ -1,9 +1,19 @@
+boto==2.48.0
+cssselect==1.0.1
+cssutils==1.0.2
+dj-database-url==0.4.2
+Django==1.10
 django-appconf==1.0.1
+django-compressor==2.1
 django-ses-backend==0.1.1
-Django==1.10
-django_compressor==2.1
 djmail==0.11.0
 futures==3.0.3
+gunicorn==19.7.1
+lxml==4.1.0
 premailer==2.9.6
-psycopg2==2.6.1
+psycopg2==2.6.2
```

Comment:
This needs to be upgraded further the psycopg2 version still breaks when installing